site/blog/OIS_Sessions_Blog

```diff
@@ -0,0 +1,53 @@
+---
+title: StarlingX sessions from Open Infrastructure Summit
+author: jeff-gowan
+date: 2019-05-22
+category: news
+---
+Did you miss out on seeing some of the StarlingX sessions at Open Infrastructure Summit? If so, don’t worry about it because this blog has you covered. 
+
+StarlingX was a hot topic at OIS this year. There were more than 15 sessions that included StarlingX in some manner. The project even got some good coverage in the keynote from Intel’s Melissa Evers-Hood. When she displayed a logo chart of contributors she shared an anecdote about one of them. She said one of the contributors told her, “I was going to do a competitive analysis of StarlingX…vs. the other projects that provide a hardened edge cloud…then realized [there are no comparisons](https://www.youtube.com/watch?v=lRR3mJrg6t8&feature=youtu.be&t=343).” This is a great illustration that helps explain why StarlingX was so popular at the event. 
+
+Fortunately, the team at OIS recorded a bunch of the sessions. Here are 5 of them for you to view on-demand. As you view these we invite you to dig in to the topics that you find most interesting. Here’s a couple of ways you can join the conversation:
+- Join the mailing list
+- Join one of the [weekly calls](https://wiki.openstack.org/wiki/Starlingx/Meetings)
+- Check out the [StarlingX Wiki page](https://wiki.openstack.org/wiki/StarlingX#Welcome_to_the_StarlingX_Project) where you can
+- Download the code
+- Read the [storyboard](https://storyboard.openstack.org/#!/project_group/86) to see what’s being worked on
+- Roll up your sleeves and [contribute bug fixes](https://bugs.launchpad.net/starlingx)
+
+Now here are the sessions recorded for you. Enjoy
+
+[StarlingX: Hardened Managed Kubernetes Platform for the Edge](https://www.youtube.com/watch?v=Wg095dkagBc&feature=youtu.be)
+
```

Comment:
I think the text blocks under the presentation links would read better if the text is a summary of the presentation as opposed to an abstract.

Reply:
The below text is also inconsistent on upper case first letters for project names.

Reply:
Summary vs. abstract: This may be true but i'm using the abstract as a proxy for a summary in lieu of reviewing all the sessions and attempting to summarize. 
I tried to catch the project names and get them in upper case, please review in case i missed something.

Reply:
I meant to reword the abstracts to sound a bit more as a summary so the post reads as an article as opposed to a list of items.